chain/gen/genesis/miners.go

```diff
@@ -217,9 +220,12 @@ func SetupStorageMiners(ctx context.Context, cs *store.ChainStore, sroot cid.Cid
 		}
 
 		err = vm.MutateState(ctx, builtin.RewardActorAddr, func(sct cbor.IpldStore, st *reward.State) error {
-			st = reward.ConstructState(qaPow)
+			*st = *reward.ConstructState(qaPow)
```

Comment:
I'm not sure why this wasn't causing more issues. Is this code unnecessary?

Reply:
And my question has been answered!

https://github.com/filecoin-project/lotus/pull/3185